.github/ISSUE_TEMPLATE/RELEASE.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Bump version in `package.json` to `X.Y.Z`:
+```bash
+$ yarn version --no-git-tag-version --new-version X.Y.Z
```

Comment:
Yarn will create an annotated tag with the `v` prefix by default. We suppress this by using `--no-git-tag-version`, allowing Git Flow tooling to take over the tagging process.